ttnn/cpp/ttnn/operations/sliding_window/sliding_window.hpp

```diff
@@ -51,10 +51,12 @@ struct SlidingWindowConfig {
     uint32_t num_cores_c = 1;                                               // num cores along width c
     CoreRangeSet core_range_set = CoreRangeSet(CoreRange({0, 0}, {0, 0}));  // active cores
 
+    int32_t divisor_override = 0;
```

Comment:
Why were these added to sliding windows, as they don't seem to used in any of the slicing_widnows impl files?